src/pages/achievement/subcomponents/AchievementDashboard.tsx

```diff
@@ -92,22 +99,25 @@ function Dashboard(props: DispatchProps & StateProps) {
       : fetchAssessmentOverviews();
 
     getAchievements();
+    fetchAssessmentConfigs();
```

Comment:
apparently, this endpoint is for staff and admin only...

but it's ok not to fetch this, since a page reload fetches the latest info from the backend anyway - can add this as a comment somewhere for FYI